### PR TITLE
chore: change all version suffix to beta-rc

### PR DESCRIPTION
**Pull request by steveeJ**
### TODO:
- [ ] [dry-run](https://github.com/holochain/holochain/actions/runs/4416441960)
- [ ] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: crates/holochain/CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
-  - Detail: any time an app is uninstalled, if the removal of that app's cells would cause there to be no cell installed which uses a given DNA, the databases for that DNA space are deleted. So, if you have an app installed twice under two different agents and uninstall one of them, no data will be removed, but if you uninstall both, then all local data will be cleaned up. If any of your data was gossiped to other peers though, it will live on in the DHT, and even be gossiped back to you if you reinstall that same app with a new agent.
+  - Detail: any time an app is uninstalled, if the removal of that app’s cells would cause there to be no cell installed which uses a given DNA, the databases for that DNA space are deleted. So, if you have an app installed twice under two different agents and uninstall one of them, no data will be removed, but if you uninstall both, then all local data will be cleaned up. If any of your data was gossiped to other peers though, it will live on in the DHT, and even be gossiped back to you if you reinstall that same app with a new agent.
```
Comment (jost-s) [inline crates/holochain/CHANGELOG.md:11]: Paul is changing these accents to single quotes in the developer docs.

**Review by jost-s — APPROVED** [review]

**Comment by jost-s** [comment]
oops, that was quicker than the speed of light

**Comment by jost-s** [comment]
The version change worked, and there's no sign of an `alpha` here https://github.com/holochain/holochain/actions/runs/4416441960/jobs/7740736165#step:9:221

**Comment by steveeJ** [comment]
> oops, that was quicker than the speed of light

it was auto-merged :smile:

**Pull request merged**